python-frontend/src/main/java/org/sonar/plugins/python/api/SubscriptionContext.java

```diff
@@ -42,6 +45,12 @@
 
   PythonFile pythonFile();
 
+  /**
+   * Returns externally declared symbols available in this context.
```

Comment:
I would something like:
`Returns symbols declared in stub files (e.g. typeshed) used in the analyzed project.`